4337/src/utils/userOp.ts

```diff
@@ -187,3 +194,36 @@ export const getSupportedEntryPoints = async (provider: ethers.JsonRpcProvider):
   console.log({ supportedEntryPoints })
   return supportedEntryPoints.map(ethers.getAddress)
 }
+
+export const encodeSignatureTimestamp = (validUntil: BigNumberish, validAfter: BigNumberish) => {
+  // Ensure that the values don't exceed 48 bits
+  if (BigInt(validAfter) > 0xffffffffffff || BigInt(validUntil) > 0xffffffffffff) {
```

Comment:
Doesn’t this need to be 
```suggestion
  if (BigInt(validAfter) > 0xffffffffffffn || BigInt(validUntil) > 0xffffffffffffn) {
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

> The boolean-returning operators allow mixing numbers and BigInts as operands:
> 
> [Relational operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#relational_operators) and [equality operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#equality_operators): >, <, >=, <=, ==, !=, ===, !==
> [Logical operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#binary_logical_operators) only rely on the [truthiness](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) of operands

Reply:
I’m in the lucky 10000 today! https://xkcd.com/1053/

Reply:
Honestly, it was a copilot-generated code and I didn't know that myself.